src/constants/message.js

```diff
@@ -0,0 +1,16 @@
+const MESSAGE = {
+  OUTPUT: {
+    GAME_START: '다리 건너기 게임을 시작합니다.\n',
+    GAME_RESULT: '최종 게임 결과',
+    MAP: (map) => `[ ${map[0].join(' | ')} ]\n[ ${map[1].join(' | ')} ]\n`,

```

Comment:
```suggestion
    MAP: ([up, down]) => `[ ${up.join(' | ')} ]\n[ ${down.join(' | ')} ]\n`,
```
요렇게 미리 풀어리시는 것도 좋을것 같아요!:)